.gitignore

```diff
@@ -77,6 +77,13 @@ coverage.xml
 *.meta.json
 *.parquet
 
+# Images
+*.jpg
+*.jpeg
+*.png
+*.gif
+*.bmp
+
```

Comment:
Hey @justinxzhao - this is a logical change. The only caveat here is that some of the examples in `ludwig/examples` have sample png images for different plots. It's not a problem for now, but I guess it'll be an issue when we try to pick up changes to those png images (if we make changes to them in the future)?

Reply:
That's a fair point. Maybe the better change would be to change the twitter bots example to download images to the `downloads/` subdirectory, which is already in our `.gitignore`. I'll give that a try.